usr/share/rear/rescue/DP/default/450_prepare_omni_systemd.sh

```diff
@@ -0,0 +1,17 @@
+# 450_prepare_omni_systemd.sh
+# Make sure Data Protector INET gets included in the rescue image (if systemd is used)
+
+if [ -r "/usr/lib/systemd/system/omni.socket" ]; then
```

Comment:
Only as a side note:
If you like to skip a whole script unless a certain condition is fulfilled
you could "Return early, return often" cf.
https://github.com/rear/rear/wiki/Coding-Style
so instead of
```
if [ -r "/usr/lib/systemd/system/omni.socket" ]; then
...
fi
```
you could return early via
```
# Nothing to do when systemd is not used:
test -r "/usr/lib/systemd/system/omni.socket" || return 0
```